Gopkg.toml

```diff
@@ -17,7 +17,12 @@ required = [
   "knative.dev/test-infra/scripts",
   "knative.dev/test-infra/tools/dep-collector",
   # For load testing.
-  "github.com/tsenart/vegeta"
+  "github.com/tsenart/vegeta",
```

Comment:
Could you change this comment to "Vegeta is for load testing"? (Right now it looks like everything below is load testing)